pkg/providers/aws/provider_postgres.go

```diff
@@ -974,6 +974,16 @@ func (p *PostgresProvider) exposePostgresMetrics(ctx context.Context, cr *v1alph
 	} else {
 		resources.SetMetric(resources.DefaultPostgresAvailMetricName, genericLabels, 1)
 	}
+
+	// cloud watch only provides us with free storage space, we need to expose more metrics to allow for more accurate alerting
+	// as `predict_linear` is hard to predict on non-linear growth & results in false positives
+	// we should follow the approach AWS take to auto scaling, and alert when free storage space is less than 10%
+	if instance != nil && instance.MaxAllocatedStorage != nil && instance.AllocatedStorage != nil{
```

Comment:
Might need a `go fmt` here? I'm surprised that isn't in our CI

Reply:
@aidenkeating we noticed last week it's not working as expected. There's a ticket to fix it: https://issues.redhat.com/browse/INTLY-9497

Reply:
Oof 👍